src/BlockchainTestsFiller/bcRandomBlockhashTest/randomStatetest70BCFiller.json

```diff
@@ -51,32 +51,6 @@
 				}
 			    }
 		   }
-	   },
-	   {
-		    "network" : ["Byzantium"],
```

Comment:
I don't have any particular expectations for this test.  Moreover, many `randomStatetest`s do not have expectations.  Somehow, `randomStatetest` numbered up to 100 have expectations, but I don't think we need that.